Pipfile

```diff
@@ -11,6 +11,7 @@ flask-script = "*"
 grpcio = "<1.28"
 grpcio-tools = "<1.28"
 gunicorn = "*"
+markupsafe = "==2.1.0"
```

Comment:
Should be downgraded below 2.1.0 to be sure that method exists based on the changelog? Because current version in stage is the one with version 2.1.0 based on Pipfile.lock. wdyt?

Reply:
Right sorry about the confusion 👍🏻 Changed in https://github.com/thoth-station/user-api/pull/1681/commits/291a587c24622cdb41afed0962166de50fb8316a